### ISSUE TITLE
brothers in arms d-day low fram rate

### ISSUE THREAD

**Issue by Levan7** [hrydgard/ppsspp#13547]
Using v1.10.3-917-gb8e832588 build x64 windows

The problem i noticed in this game is that the frame rate jumps from 50-60 however i can't blame the system because when i press turbo mode it always speeds up over 200 fps

even enable skip frame notification popped up so i found this strange with this game, feels like the game slows down when played.

[logs.zip](https://github.com/hrydgard/ppsspp/files/5380351/logs.zip)

**Comment by ANR2ME** [comment]
@Levan7 you'll need to provide more info for the devs to reproduce the issue, like what kind of hardware/device you're using and your Operating System, also Graphics settings.

Currently there is a report that OpenGL on AMD GPU drivers on Win10 x64 having FPS drops on Gran Turismo, so may be you also having similar issue?

**Comment by MojoJojoDojo** [comment]
> @Levan7 you'll need to provide more info for the devs to reproduce the issue, like what kind of hardware/device you're using and your Operating System, also Graphics settings.
> 
> Currently there is a report that OpenGL on AMD GPU drivers on Win10 x64 having FPS drops on Gran Turismo, so may be you also having similar issue?

TBH OpenGL should never be used with AMD GPUs in the first place because their driver is not good and optimized at all.
So its mostly a driver issue perhaps.

**Comment by Levan7** [comment]
I'm using nvidia 1050ti with dx11 api 
this is what i'm talking about the frame rate is 50-60 in most locations but when i press the tubo button it speeds up

https://imgur.com/SRQBeP9

also one strange thing i noticed when i enabled ppsspp in emulator speed/fps it always shows around 100% (99.6%) even though the frame rate and audio is choppy.

**Comment by unknownbrackets** [comment]
Could you enable "Show debug statistics", and then take a few screenshots?  I just want to confirm whether any part is running slow.

-[Unknown]

**Comment by Panderner** [comment]
![Screenshot_2021-02-21-14-24-36-562_org ppsspp ppsspp](https://user-images.githubusercontent.com/37503397/108617343-9de26e00-7450-11eb-828e-fa77c72c0b13.jpg)

Here's a screenshot

**Comment by unknownbrackets** [comment]
Has performance changed or improved at all?  There have been a few optimizations, though moreso in the Vulkan code.

A GE frame dump (from a recent git build) would also help.

-[Unknown]

**Comment by Gamemulatorer** [comment]
No feedback/outdated?

**Comment by ppmeis** [comment]
Tested in latest build. Game still even using vulkan:

![image](https://user-images.githubusercontent.com/4381277/226734893-8507a094-1a00-483f-8b52-a13a6104d43b.png)

[ULES00608_0001.zip](https://github.com/hrydgard/ppsspp/files/11033397/ULES00608_0001.zip)

**Comment by Virjoinga** [comment]
> Tested in latest build. Game still even using vulkan:
> 
> ![image](https://user-images.githubusercontent.com/4381277/226734893-8507a094-1a00-483f-8b52-a13a6104d43b.png)
> 
> [ULES00608_0001.zip](https://github.com/hrydgard/ppsspp/files/11033397/ULES00608_0001.zip)

Yeah I have the same problem, I am even using an RTX 3080, Ryzen 9 7950X and 32GB of RAM. Will not hit a constant 60fps.

**Comment by unknownbrackets** [comment]
Okay, I was concerned about the 17ms display list processing time, but it really sounds like this game is suffering from one of:
 * Poor estimation of GPU cycle costs
 * Poor estimation of CPU/HLE cycle costs
 * Detection issues in flips

Although the bloom effect in the frame dump is a bit weird, I don't see anything horrible happening.

One thing that would be interesting: if you enable the software renderer (especially on a 7950X), does it run any more consistently?  This should be heavy on the CPU and won't look as good, but it might help confirm there's not something else going on here.

-[Unknown]

**Comment by AaronJk12378** [comment]
Is this problem fix? I've download the game, and it runs 40-60 fps. But the sound is "choppy and delayed", can someone provide a 30fps hack for it? Same thing as GOW on PPSSPP?
